civis/base.py

```diff
@@ -103,7 +105,7 @@ class Endpoint:
 
     _lock = threading.Lock()
 
-    def __init__(self, session_kwargs, client, return_type='civis'):
+    def __init__(self, session_kwargs, client, return_type="raw"):
```

Comment:
Fixing this hidden bug that never got exposed. This `return_type` is for the `civis.response.Response` objects, where {"raw", "snake"} are the only options. We've never run into an exception of an unrecognized return type "civis", because this `Endpoint` class is only used dynamically (from [here](https://github.com/civisanalytics/civis-python/blob/6e18575234ecf5bdca2e801ab710041b312146c9/civis/resources/_resources.py#L472) onwards) to generate the individual endpoint classes. These endpoint class instances are then created when a `civis.APIClient` instance is created, where `return_type` is set to "snake" by default ([here](https://github.com/civisanalytics/civis-python/blob/a884416673a358436703732aa1b8127bbde48213/civis/civis.py#L387)).

Reply:
Good catch!